toolkit/tools/pkg/imagecustomizerlib/imagecreator.go

```diff
@@ -85,38 +86,41 @@ func doOsCustomizationsImageCreator(
 		return err
 	}
 
+	ctx := context.Background()
```

Comment:
Any chance you can plumb the `ctx` variable all the way from the top-level `main` functions?

This is more morally correct. Also, it will make it easier to add Ctrl+C support in the future.